package.json

```diff
@@ -27,24 +27,24 @@
   "main": "build/index.js",
   "description": "A specification backed popover for react",
   "devDependencies": {
-    "react-hot-loader": "^1.2.4",
-    "babel-core": "^4.7.16",
     "babel": "^5.0.0",
-    "coffee-script": "^1.9.1",
-    "chai": "^2.2.0",
-    "webpack": "^1.7.3",
-    "style-loader": "^0.9.0",
-    "bootstrap-webpack-plugin": "^0.2.1",
+    "babel-core": "^4.7.16",
+    "babel-loader": "^4.2.0",
     "bluebird": "^2.9.24",
+    "bootstrap-webpack-plugin": "^0.2.1",
+    "chai": "^2.2.0",
     "coffee-loader": "^0.7.2",
-    "react-tappable": "^0.5.1",
-    "babel-loader": "^4.2.0",
-    "ramda": "^0.14.0",
-    "webpack-dev-server": "^1.8.0",
+    "coffee-script": "^1.9.1",
     "css-loader": "^0.9.1",
     "object.assign": "^1.1.1",
+    "ramda": "^0.14.0",
     "react": "^0.13.1",
-    "react-draggable": "^0.4.0"
```

Comment:
Can you explain this. My scan did not catch any other version changes but please confirm in case I'm wrong. Thanks.